apps/wallet/src/features/Send/InputTransaction/CollectiblesTab.tsx

```diff
@@ -65,11 +93,21 @@ export const CollectiblesTab: FC<Props> = ({ onContinue }) => {
 				selectedItemIconStyle={styles.selectedNftIcon}
 			/>
 
-			<RecipientInput />
+			<CheckedInput
+				value={receiver}
+				placeholder="Recipient account"
+				errorText={recipientInput.message}
```

Comment:
Same topic with about, the `recipientInput.message` is only used for this component, try to keep things abstract, and separate.